R/create_episode_file.R

```diff
@@ -159,7 +160,9 @@ create_episode_file <- function(
         hc_cost_q4 = NA,
         hc_provider = NA,
         hc_reablement = NA,
-        sds_option_4 = NA,
```

Comment:
has sds_option 4 been removed?

Reply:
It is coded in smrtype instead with the other 3 options so i removed this from here